src/Control/Foldl.hs

```diff
@@ -1048,6 +1049,17 @@ handles k (Fold step begin done) = Fold step' begin done
     step' = flip (appEndo . getDual . getConst . k (Const . Dual . Endo . flip step))
 {-# INLINABLE handles #-}
 
+{- | @{foldOfWith f folder xs} folds all values from a Lens, Traversal, Prism or Fold with the given folder
+
+>>> foldOfWith f folder xs == L.fold folder (xs^..f)
```

Comment:
Use a single `>` instead of `>>>`

`>` is used for arbitrary fixed-width code whereas `>>>` is specially designed example REPL input/output pairs, like:

```
>>> 2 + 2
4
```

Also, `doctest` integrates with `>>>` to test that the command produces the expected output